lessonFour.py

```diff
@@ -1,36 +1,50 @@
 # complete this function so that it prints Hello, World
 def printHelloWorld() :
-    # your code here
+    print("Hello, World")
 
 # complete this function so that it prints whatever is provided in the parameter
 def printParam(param) :
-    # your code here
+    print(param)
 
 # complete this function so that it returns the sum of num1 and num2
 def sumTwoNumbers(num1, num2) :
-    # your code here
+     sum = num1 + num2
+
+     return sum
 
 # complete this function so that it returns the square of the num parameter
 def findSquare(num) :
-    # your code here
+    sum = num * num
+
+    return sum
 
 # complete this function so that it returns True if day is Wednesday
 # otherwise return False
 def dayIsWednesday(day) :
-    # your code here
+    if day == "Wednesday" :
+        return True
+    else :
+        return False
+
 
 
 myList = [ "mainsheet", "batton", "rudder", "foil", "block" ]
 
 # complete this function so that it return "I got it" if item is in myList
 # otherwise return "I have to get it"
-def findInList(item) :
-    # your code here
+def findInList(finditem) :
+    for item in myList :
+        if finditem == item :
+            return "I got it"
+
+    return "I have to get it"
 
 # assuming every element in 'numList' is a number, complete this function so that it
 # returns the total sum of the elements in the list
 def complexSum(numList) :
-    # your code here
+    sum = numList
```

Comment:
This one isn't quite right. You have to loop through numList, otherwise you're just declaring `sum` as a list.